karma.conf.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function(config) {
+    config.set({
+        files: [
+            'bower_components/underscore/underscore.js',
```

Comment:
Looks like you removed underscore from the project but not from the karma configuration.